package.json

```diff
@@ -5,7 +5,7 @@
   "private": true,
   "engine": "node >= 4.2",
   "dependencies": {
-    "bower": "1.7.1",
+    "bower": "1.8.0",
     "del": "1.1.1",
```

Comment:
This change shouldn't be propagated to production. We upped this version to force the admin-fe to work on your local environment rather than for some need based on the repo as a whole.